docs/UserGuide.md

```diff
@@ -138,8 +188,7 @@ Format: `add n/NAME s/STUDENT_ID e/EMAIL h/TELEGRAM_HANDLE [l/WEBLINK] [c/TIMETA
 </box>
 
 Examples:
-* `add n/John Doe s/A1234567X h/@john.doe e/johnd@example.com t/tutorial-1 t/high-ability c/mon: 8-10, 10-12 tue:
-  11-13 thu: 12-15, 15-17`
+* `add n/John Doe s/A1234567X h/@john.doe e/johnd@example.com l/https://github.com/johndoe/tp t/tutorial-1 t/high-ability c/mon: 8-10, 10-12 tue: 11-13 thu: 12-15, 15-17`
```

Comment:
Just a heads up all john.doe will be changed to john_doe as it '.' is not allowed in telegram handles

Reply:
I have changed all existing john.does in my PR, but i hope yours wont override it back